models/Pokemon.go

```diff
@@ -0,0 +1,56 @@
+package models
+
+import (
+	"strconv"
+	"strings"
+)
+
+// Pokemon represet the basic Pokemon model
+type Pokemon struct {
+	ID             int    `json:"id"`
+	Name           string `json:"name"`
+	TypeOne        string `json:"typeOne"`
+	TypeTwo        string `json:"typeTwo"`
+	Total          int    `json:"total"`
+	HealthPoints   int    `json:"healthPoints"`
+	Attack         int    `json:"attack"`
+	Defense        int    `json:"defense"`
+	SpecialAttack  int    `json:"specialAttack"`
+	SpecialDefense int    `json:"specialDefense"`
+	Speed          int    `json:"speed"`
+	Generation     int    `json:"generation"`
+	Legendary      bool   `json:"legendary"`
+}
+
+// PokemonFromString return a Pokemon from a string
+func PokemonFromString(csvRow string) (p Pokemon, err error) {
+	pokeRow := strings.Split(string(csvRow), ",")
```

Comment:
No need to convert `csvRow` to string; it's already an string.
```suggestion
	pokeRow := strings.Split(csvRow, ",")
```